--- ./source/test/CMakeLists.txt.orig	2016-11-08 00:20:30.255328576 +0000
+++ ./source/test/CMakeLists.txt	2016-11-08 00:16:38.709703855 +0000
@@ -44,7 +44,7 @@ add_executable(TestBench ${YASM_SRC}
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
 
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     if(EXTRA_LIB)
         list(APPEND LINKER_OPTIONS "-L..")
