diff -up multicoreware-x265-d6257335c537/source/CMakeLists.txt.r multicoreware-x265-d6257335c537/source/CMakeLists.txt
--- multicoreware-x265-d6257335c537/source/CMakeLists.txt.r	2014-07-09 10:25:15.000000000 +0200
+++ multicoreware-x265-d6257335c537/source/CMakeLists.txt	2014-07-10 12:03:21.459088376 +0200
@@ -308,7 +308,7 @@ if(X265_LATEST_TAG)
     # Produce a pkg-config file
     configure_file("x265.pc.in" "x265.pc" @ONLY)
     install(FILES       "${CMAKE_CURRENT_BINARY_DIR}/x265.pc"
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/pkgconfig")
+            DESTINATION "${LIB_INSTALL_DIR}/pkgconfig")
 endif()
 
 if(NOT WIN32)
